src/profiler.h

```diff
@@ -26,6 +26,14 @@
 #include "writer.h"
 
 
+# ifndef likely
+#  define likely(x)     (__builtin_expect(!!(x), 1))
+# endif
+
+# ifndef unlikely
+#  define unlikely(x)   (__builtin_expect(!!(x), 0))
+# endif
```

Comment:
`profiler.h` is probably not the right place for these macros, they are only used in `mallocTracer.cpp`, you can put them there. Or, to make them available for a wider use, `arch.h` will fit better.